utilities/pipelines/resourcePublish/Get-ModulesToPublish.ps1

```diff
@@ -31,13 +31,13 @@ function Get-ModifiedFileList {
         [string] $CompareCommit = 'HEAD^'
     )
 
-    Write-Verbose "Gathering modified files between [$Commit] and [$CompareCommit]" -Verbose
+    Write-Verbose "Gathering modified files between [$CompareCommit] and [$Commit]" -Verbose
     $Diff = git diff --name-only --diff-filter=AM $CompareCommit $Commit
-    $ModifiedFiles = $Diff | Get-Item
-    Write-Verbose 'The following files have been added or modified:' -Verbose
-    $ModifiedFiles | ForEach-Object {
-        Write-Verbose (' - [{0}]' -f $_.FullName) -Verbose
-    }
+    $ModifiedFiles = $Diff | Get-Item -Force
+    # Write-Verbose 'The following files have been added or modified:' -Verbose
```

Comment:
Remove comments

Reply:
Will remove comments, to reduce the verbosity of the logging as discussed earlier.

Reply:
Done